### PR TITLE
Rec play performance

### PR DESCRIPTION
**Pull request by DavidLesnjak**

### CODE REVIEW

File: sds/config/sds_play_config.h [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 Arm Limited. All rights reserved.
+ * Copyright (c) 2024 Arm Limited. All rights reserved.
```
Comment (RobertRostohar) [inline sds/config/sds_play_config.h:2]: 2023-2024

File: sds/config/sds_rec_config.h [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 Arm Limited. All rights reserved.
+ * Copyright (c) 2024 Arm Limited. All rights reserved.
```
Comment (RobertRostohar) [inline sds/config/sds_rec_config.h:2]: 2023-2024

File: sds/source/sds_play.c [hunk 3]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2023 Arm Limited. All rights reserved.
+ * Copyright (c) 2024 Arm Limited. All rights reserved.
```
Comment (RobertRostohar) [inline sds/source/sds_play.c:2]: 2023-2024

**Review by RobertRostohar — COMMENTED** [review]
Looks fine. See inline comments for disclaimer.

File: sds/config/sds_play_config.h (hunk 1 above)
Comment (DavidLesnjak) [reply sds/config/sds_play_config.h]: Updated disclaimer is force-pushed.

Comment (DavidLesnjak) [reply sds/config/sds_play_config.h]: Updated disclaimer is force-pushed.

File: sds/source/sds_play.c (hunk 3 above)
Comment (DavidLesnjak) [reply sds/source/sds_play.c]: Updated disclaimer is force-pushed.

**Pull request merged**